.storybook/webpack.config.js

```diff
@@ -21,30 +22,31 @@ module.exports = async ({ config }) => {
         {
             test: /\.tsx?$/,
             exclude: /node_modules/,
-            use: [
-                { loader: 'babel-loader' },
-                { loader: 'react-docgen-typescript-loader' }
-            ]
+            use: [{ loader: 'babel-loader' }, { loader: 'react-docgen-typescript-loader' }],
+        },
+        {
```

Comment:
This is the new chunk? Doesn't this map to `.j, js, .t, .ts`? The `.tsx?` above maps to `.ts, .tsx`

Reply:
I can scope it to only map files, the reason is we want to use Storybook to aid in development for annotations with ContentPreview and sourcemaps from annotations and preview js will aid in that effort.

Reply:
Isn't it _excluding_ annotations right now though?

Reply:
Not sure what the webpack parse order is but it looks like something else is handing *.js in the base config: ttps://github.com/box/box-ui-elements/blob/master/scripts/webpack.config.js#L151

Reply:
No it is excluding everything in node_modules besides annotations.

Reply:
I am also not looking to include source maps in the release or anywhere else besides storybook.